### ISSUE TITLE
Adjustable regularization

### ISSUE THREAD

**Issue by deiruch** [dotnet/machinelearning-modelbuilder#2370]
**Is your feature request related to a problem? Please describe.**
I know that my training set for a classification problem has only 80% accurate labels. Today the models produced by modelbuilder are trying to overfit. I'd like to train a regularized model that doesn't overfit.

**Describe the solution you'd like**
I would like the ability to control the regularization strength/type used to train the model.

**Describe alternatives you've considered**
None

**Additional context**
-

**Comment by beccamc** [comment]
@luisquintanilla Framework or Model Builder ask?

**Comment by luisquintanilla** [comment]
Thanks @deiruch for this request.

@beccamc this should already be possible in the framework for some algorithms. I could see this setting being part of advanced training options. Let's check whether we've received similar requests in the past and monitor for additional feedback.
